stable/aws-pod-identity-webhook/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+appVersion: "c0431e1"
```

Comment:
> Did you want the above specific Git commit for some reason?

Yes because this PR (including the identical previous PR) has dragging on over a year.

Reply:
Edit: there isn't proper tags pushed to docker hub: https://hub.docker.com/r/amazon/amazon-eks-pod-identity-webhook/tags